src/test/java/smarthome/controllers/SimulationParametersTest.java

```diff
@@ -0,0 +1,61 @@
+package smarthome.controllers;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import smarthome.models.User;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SimulationParametersTest {
+
+    @Test
+    void addUser() {
```

Comment:
You might want to work on giving tests more meaningful name. Naming tests properly is actually a huge point of contention.

Again, this is not important right now, so it passes.